ohtu2021-kvantti/WebMark
# removed init stuff

## summary
Removed `admin`, `django.contrib`, `from` and `import` from /WebCLI. These were most likely left there during init 26 days ago.
## How to test
Check if building from scratch doesn't crash anything. Also check if creating an algorithm doesn't crash.
